app/javascript/packs/drawerSliders.js

```diff
@@ -40,4 +40,6 @@ function initializeDrawerSliders() {
       InstantClick.display(url);
     });
   }
-}
+};
```

Comment:
I can't actually find a view with `feed-filter-select` anywhere in the app - is this code still necessary?

Reply:
For what it's worth, it looks like it [used to be part of `articles/index`](https://github.com/ludwiczakpawel/dev.to/blob/e52f21afcce9a2d9f6bb1349554c6c4303bb4095/app/views/articles/index.html.erb#L58) and [`tagged_articles/index`](https://github.com/kuei0221/forem/blob/b07e668617e0ed2a58ff5b5bccead924ce8e44ea/app/views/stories/tagged_articles/index.html.erb#L79), was removed from `articles/index` in [this PR](https://github.com/forem/forem/pull/14738) and left `articles/tagged/index` in [this PR](https://github.com/forem/forem/pull/15435) — I _think_ it's safe to remove it!